.evergreen/config.yml

```diff
@@ -2795,7 +2795,7 @@ buildvariants:
 
 - matrix_name: "tests-windows-enterprise-auth"
   matrix_spec:
-    platform: windows-64-vsMulti-small
```

Comment:
windows-64-vsMulti-small is/was a special distro that was built just for us to test with. I take it we don't need that anymore?

Reply:
https://jira.mongodb.org/browse/BUILD-9934

Ah it was for testing python <=3.4. So yea we don't need it anymore.

Reply:
This is just changing the name we give it, not the actual host